media-video/fuppes/fuppes-1.0.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="Free UPnP A/V media server"
+DESCRIPTION="FUPPES is a free, multiplatform UPnP A/V Media Server. \
+FUPPES supports a wide range of UPnP MediaRenderers as well as on-the-fly \
+transcoding of various audio, video and image formats. FUPPES also includes \
+basic DLNA support."
+HOMEPAGE="https://github.com/uvoelkel/fuppes/"
+COPYRIGHT="2007-2009 Ulrich Völkel"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="1c9b4cc78e2dd3f424face74d06ac3918ea38180"
+SOURCE_URI="https://github.com/uvoelkel/fuppes/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="b30388408591ebdd71e3b9334d9efafa5fb53330e2938127b995c0f4330d2470"
+SOURCE_DIR="fuppes-$srcGitRev"
+PATCHES="fuppes-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	var/lib/fuppes directory keep-old
+	"
+
+PROVIDES="
+	fuppes$secondaryArchSuffix = $portVersion
+	cmd:fuppes$secondaryArchSuffix = $portVersion
+	cmd:fuppesd$secondaryArchSuffix = $portVersion
+	lib:libfuppes$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libavformat$secondaryArchSuffix
+	lib:libavutil$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libev$secondaryArchSuffix
+	lib:libevent$secondaryArchSuffix
+	lib:libexiv2$secondaryArchSuffix
+	lib:libfaad$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libmad$secondaryArchSuffix
+	lib:libMagickWand_7.Q16HDRI$secondaryArchSuffix
```

Comment:
Maybe lib:libMagickWand_6.Q16 (from what I see lib:libMagickWand_7.Q16HDI doesn't provide the headers?)